src/main/java/frc/robot/subsystems/CommandSwerveDrivetrain.java

```diff
@@ -70,8 +74,10 @@ public class CommandSwerveDrivetrain extends TunerSwerveDrivetrain implements Su
   CurrentLimitsConfigs m3_current_config = new CurrentLimitsConfigs();
   CurrentLimitsConfigs m4_current_config = new CurrentLimitsConfigs();
 
-  public PIDController PID_X = new PIDController(1.5, 0.01, 0.1);
-  public PIDController PID_Y = new PIDController(1.5, 0.01, 0.1);
+  public Constraints constraints = new TrapezoidProfile.Constraints(5, 4);
```

Comment:
Do we need these constants to be explained better?  Given a name?
I'm guessing the values should be the same for the X and Y PIDs generally